### PR TITLE
Bug Fix 

### PR DESCRIPTION
**Pull request by kojiadrianojr**
* Default logo no display
* Token logo no display
* Update token list 
* Update Srk Logo
* MetaData temporary information
* Add information on config with SparkPoint socmeds and token info
* Add public image srk and srkdefi

### CODE REVIEW

**Review by aldrickb — APPROVED** [review]
LGTTM! Pero wait natin si sir Rico regarding sa metadata if good na si mga phrases.

**Review by swengr-janan — CHANGES_REQUESTED** [review]
Master Koj, naka redirect pa to sa pancake swap medium site image

**Review by swengr-janan — APPROVED** [review]
Confirmed!

**Review by Pseudoman21 — APPROVED** [review]
Aligned!

**Review by harveyjavier — APPROVED** [review]
Enhanced!

File: public/index.html [hunk 1]
```diff
@@ -27,15 +27,15 @@
...
       name="description"
-      content="Flip CAKE, collect SYRUP and stake them with Pancake Swap, the best decentralized exchange on Binance Smart Chain."
+      content="Refuel your rocket with SFUEL and travel to the moon with SparkSwap, the G.O.A.T decentralized exchange on Binance Smart Chain."
     />
     <meta name="theme-color" content="#000000" />
     <meta name="fortmatic-site-verification" content="j93LgcVZk79qcgyo" />
     <meta name="viewport" content="initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=no" />
-    <meta name="twitter:image" content="https://pancakeswap.finance/hero.png" />
-    <meta name="twitter:description" content="Farm the native coin of PancakeSwap with PancakeSwap FLP Token" />
+    <meta name="twitter:image" content="https://srk.finance/" />
```
Comment (ricoz) [inline public/index.html:35]: Link an image, pwede yung SparkDeFi logo

File: public/index.html [hunk 2]
```diff
@@ -27,15 +27,15 @@
...
+    <meta name="twitter:description" content="Farm the native coin of SparkSwap with SparkSwap FLP Token" />
```
Comment (ricoz) [inline public/index.html:36]: Change description to same as content above (Yung may "Refuel your rocket...")

File: public/index.html [hunk 3]
```diff
@@ -27,15 +27,15 @@
...
     <meta name="twitter:card" content="summary" />
-    <meta name="twitter:title" content="🥞 PancakeSwap - A next evolution DeFi exchange on Binance Smart Chan (BSC)" />
+    <meta name="twitter:title" content="🚀 SparkSwap - A new way to defy DeFi exchange on Binance Smart Chan (BSC)" />
```
Comment (ricoz) [inline public/index.html:38]: Change "A new way to defi" to "The preferred"

**Review by ricoz — CHANGES_REQUESTED** [review]
Minor change requests

File: public/index.html (hunk 2 above)
Comment (kojiadrianojr) [reply public/index.html:36]: Sir palitan ko rin po yung G.O.A.T ng "Prodigious"?

File: public/index.html [hunk 4]
```diff
@@ -27,15 +27,15 @@
...
       name="description"
-      content="Flip CAKE, collect SYRUP and stake them with Pancake Swap, the best decentralized exchange on Binance Smart Chain."
+      content="Refuel your rocket with SFUEL and travel to the moon with SparkSwap, the prodigious decentralized exchange on Binance Smart Chain."
     />
     <meta name="theme-color" content="#000000" />
     <meta name="fortmatic-site-verification" content="j93LgcVZk79qcgyo" />
     <meta name="viewport" content="initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=no" />
-    <meta name="twitter:image" content="https://pancakeswap.finance/hero.png" />
-    <meta name="twitter:description" content="Farm the native coin of PancakeSwap with PancakeSwap FLP Token" />
+    <meta name="twitter:image" content="%PUBLIC_URL%/images/192x192_App_Icon.png" />
```
Comment (ricoz) [inline public/index.html:35]: Nagagana yan na `%PUBLIC_URL%`?

**Pull request merged**